odh/base/monitoring/kustomization.yaml

```diff
@@ -4,3 +4,4 @@ kind: Kustomization
 namespace: opf-monitoring
 resources:
   - kfdef.yaml
+  - dashboards/jupyterhub-dashboard.yaml
```

Comment:
thinking more on it I think it would be better to change this to `dashboards` and add a new kustomization file like so `/dashboards/kustomization.yaml` and include the yaml in there instead. This way when adding a dashboard you only need to make changes to the `dashboard` folder.

Reply:
ya that makes sense, updated it :+1: